packages/InteractiveProfilingTool-Tests.package/IPTReportMock.class/instance/model.st

```diff
@@ -0,0 +1,4 @@
+accessing
+model
+
+	^ self
```

Comment:
@Alexander-Dubrawski: Warum das?

Reply:
Das habe ich zuvor in einem Test mal verwendet. Jetzt benutze ich dies jedoch auch nicht mehr und es kann gelöscht werden.